mvc/controllers/actions.rst

```diff
@@ -1,8 +1,63 @@
-.. include:: /../common/stub-topic.txt
+Actions and Action Results
+==========================
 
-|stub-icon| Actions and Action Results
-===========================================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+Actions and action results are a fundamental part of how developers build apps using ASP.NET MVC.
 
-.. _issue: https://github.com/aspnet/Docs/issues/118
+Sections:
+	- `What is a Controller`_
+	- `Defining Actions`_
+
+`View or download sample from GitHub <https://github.com/aspnet/Docs/tree/1.0.0-rc1/mvc/performance/response-caching/sample>`_.
+
+What is a Controller
```

Comment:
I think there needs to be a statement in this section somewhere about how a request makes its way to a controller action (i.e via routing and action selection) and provide a link to one of the routing topics with more details.